### PR TITLE
cleanup code around boto

### PR DESCRIPTION
**Pull request by peterbe**
Stylistic cleanup and change from `self.assertEqual` to `eq_`.

### CODE REVIEW

File: socorro/external/boto/crashstorage.py [hunk 1]
```diff
@@ -364,19 +360,18 @@ class SupportReasonAPIStorage(BotoS3CrashStorage):
...
+    #     BotoS3CrashStorage,
+    #     'bucket_name',
+    #     'mozilla-support-reason'
+    # )
+    # required_config.prefix = change_default(
+    #     BotoS3CrashStorage,
+    #     'prefix',
+    #     'mozilla-support-reason'
+    # )
+
```
Comment (adngdb) [inline socorro/external/boto/crashstorage.py]: I suspect the comment above is wrong and we should really just delete this. :)

File: socorro/unittest/external/boto/test_crashstorage.py [hunk 2]
```diff
@@ -647,16 +642,16 @@ def temp_failure_fn(*args):
...
             mock.call("936ce666-ff3b-4c7a-9674-367fe2120408", "raw_crash"),
             mock.call("936ce666-ff3b-4c7a-9674-367fe2120408", "raw_crash"),
             mock.call("936ce666-ff3b-4c7a-9674-367fe2120408", "raw_crash"),
         ])
 
-        self.assertEqual(result, self._fake_unredacted_processed_crash())
+        eq_(result, self._fake_unredacted_processed_crash())
 
-    def test_get_unredacted_processed_crash_with_consistency_trouble_no_recover(self):
+    def test_get_unredacted_processed_crash_with_consistency_trouble_no_recover(self):  # NOQA
```
Comment (adngdb) [inline socorro/unittest/external/boto/test_crashstorage.py]: Why not just shrink down the name of that function?

**Comment by adngdb** [comment]
r+

File: socorro/unittest/external/boto/test_crashstorage.py (hunk 2 above)
Comment (peterbe) [inline socorro/unittest/external/boto/test_crashstorage.py]: Did not dare to change too much.

**Pull request merged**